src/components/organisms/UiList/_internal/UiListItemSuffixAsButton.vue

```diff
@@ -79,3 +79,18 @@ const defaultProps = computed(() => ({
 }));
 const hasIcon = computed(() => (!!defaultProps.value.iconSuffixAttrs.icon));
 </script>
+
+<style lang="scss">
+@use "../../../../styles/functions";
+
+.ui-list-item-suffix-as-button {
+  $element: list-item-suffix-as-button;
+
+  gap: functions.var($element, gap, var(--space-4));
```

Comment:
@ArkadiuszKrzymienski you have right about this approach, `gap` is awesome 🙈 . When you use `visuall-hidden on label you have always right spacing